### PR TITLE
Unit test Cases for  Revert Staking, Minting, Revert Redemption, Revert Unstaking

### PR DESCRIPTION
**Pull request by abhayks1**

### CODE REVIEW

File: contracts/OpenSTValue.sol [hunk 1]
```diff
@@ -380,20 +380,20 @@ contract OpenSTValue is OpsManaged, Hasher {
 		
 		Unstake storage unstake = unstakes[_redemptionIntentHash];
 
-    	// require that the unstake has expired and that the redeemer has not
-    	// processed the unstaking, ie unstake has not been deleted
-    	require(unstake.expirationHeight > 0);
-    	require(unstake.expirationHeight <= block.number);
+			// require that the unstake has expired and that the redeemer has not
```
Comment (benjaminbollen) [inline contracts/OpenSTValue.sol]: we will need to discuss a standard and then implement a Linter check in the CI (to be put in place); for now OK, but the editors fighting is really not clean on the origin repo

File: test/OpenSTUtility.js [hunk 2]
```diff
@@ -535,6 +536,144 @@ contract('OpenSTUtility', function(accounts) {
...
-})
+
+	// Unit test cases for revert rdemption
+	describe('revert redemption', async () => {
+
+		context('revert redemption', async () => {
+
+			var stakeAmountST = 1,
+					convertedAmountBT = (stakeAmountST*conversionRate),
+					redemptionAmountBT = 2, // How many Branded tokens to redeem
```
Comment (benjaminbollen) [inline test/OpenSTUtility.js]: This is in BT Wei, not BT; please correct to BT

File: contracts/OpenSTValue.sol (hunk 1 above)
Comment (kushal-ladha) [reply contracts/OpenSTValue.sol]: Hey Ben, me, deepesh and abhay have synched are tab space to two. We can decide on which linter to use as we all use different editors for coding.

File: test/OpenSTValue.js [hunk 3]
```diff
@@ -506,4 +508,250 @@ contract('OpenSTValue', function(accounts) {
...
+
+
+	// Revert Staking before ProcessStaking
+	describe('Process RevertStaking before ProcessStaking', async () => {
+
+		context('Revert Stake', async () => {
+
+			var staker = accounts[0];			
+			var owner  = accounts[1];
+			var amountST = 126;
```
Comment (benjaminbollen) [inline test/OpenSTValue.js]: We should do full scan of unit tests of consistent use of ST vs STWei (UT vs UTWei, BT vs BTWei)

**Comment by benjaminbollen** [comment]
closes #45 
closes #67 
closes #68 
closes #69

**Pull request merged**